plugins/scaffolder-backend/sample-templates/local-templates.yaml

```diff
@@ -10,4 +10,5 @@ spec:
     - ./create-react-app/template.yaml
     - ./springboot-grpc-template/template.yaml
     - ./v1beta2-demo/template.yaml
+    - ./rails-demo/template.yaml
```

Comment:
Not sure we should add this to the the default templates when the action isn't shipped out of the box. Let's remove this from here, as I think it's gonna cause confusion when this template fails when people try to run it.